### ISSUE TITLE
[Idea] Make beacons last longer (to make them more usefull)

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#2061]
Issue initially reported there: https://github.com/DaemonEngine/Daemon/issues/642

> Hi!
>
> The beacons appears for only 3 or 5 seconds, that is too short.
> The time to almost understand where is it, and it lost.
>
> Also, a path on ground (with jump possible?) to the nearest beacon should be show for Aliens, and Humans with helmet.
>
> Humans bots should also put beacons, when they saw Aliens buildings for example.
>
> When the building disappear, the beacon should too.
> For beacons for defend, attack... I think 30 seconds should be enough.
> -- @superdupont-fr

> Ideally we would have some lifetime management for beacons because even when "Defend" used to last 10s I often came for nothing because it was too late already
>
> EDIT: the idea would be to make the beacon disappear if the attacker is killed in case of a Defend beacon, if the author of a "time sync" > beacon is killed, etc. (Note that it's hard and I dont intend to do it)
> -- @necessarily-equal

**Comment by slipher** [comment]
We just went over beacon durations and increased some of them. https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir/pull/122

**Comment by bmorel** [comment]
Yes, all duration which were easy to adjust were done. Sadly, it does not means that all useful beacons had this problem improved.
